pages/doc/tracing_ui_overview.md

```diff
@@ -95,6 +95,17 @@ The standard dashboard for service metrics is read-only. To create a customizabl
 
 After you save the clone, you can find it by name from the **Dashboards** menu of the task bar, and you can use it to drill down to the Traces browser.
 
+### Navigate to the Tracing UI from the Service Metrics Dashboard
```

Comment:
Here I like the more wordy heading. 
But not in the release notes, where all the other headings are very succinct.